.eslintrc

```diff
@@ -33,6 +30,7 @@
     "no-mixed-operators": [0],
     "no-lonely-if": [1],
     "no-bitwise": [0],
-    "arrow-parens": [0]
+    "arrow-parens": [0],
+    "operator-linebreak": [0] // handled by Prettier
```

Comment:
I tried to keep the ESLint rules all the same, but I did make one change: I swapped out `eslint-config-airbnb` with `eslint-config-airbnb-base`. That has the same rules, but you don’t need to install all the React & JSX plugins.